api/oauth-config/index.ts

```diff
@@ -0,0 +1,14 @@
+import "../startup";
+import { Context, HttpRequest } from "@azure/functions";
+
+export default async function (context: Context, req: HttpRequest): Promise<void> {
+  context.res = {
+    status: 200,
+    body: JSON.stringify({
+      domain: process.env.AUTH0_DOMAIN,
+      clientID: process.env.AUTH0_CLIENTID,
```

Comment:
Duplicate clientID entry. Remove the first one as it doesn't have a standard structure.